packages/next/src/lib/turbopack-warning.ts

```diff
@@ -214,7 +214,17 @@ export async function validateTurboNextConfig({
       babelrc
     )})\n  ${`Babel is not yet supported. To use Turbopack at the moment,\n  you'll need to remove your usage of Babel.`}`
   }
-  if (unsupportedConfig.length) {
+
+  if (
+    unsupportedConfig.length === 1 &&
+    unsupportedConfig[0] === 'experimental.optimizePackageImports'
```

Comment:
unlike other options, there are some internal default user can't change (https://github.com/vercel/next.js/blob/canary/packages/next/src/server/config.ts#L677) so we can't adjust test config to escape this. For now, workaround by emitting warning only until turbopack supports same (WEB-1538).